app/components/file-compare/header/index.hbs

```diff
@@ -0,0 +1,154 @@
+<AkStack
+  ...attributes
+  @direction='column'
+  @width='full'
+  local-class='file-compare-header-container'
+  data-test-fileCompare-header-root
+>
+  <div
+    local-class='file-compare-header-breadcrumbs'
+    data-test-fileCompare-headerBreadcrumbs
+  >
+    {{yield to='breadcrumbs'}}
+  </div>
+
+  <AkStack @direction='column' @width='full'>
+    <AkStack
+      @width='full'
+      @justifyContent='space-between'
+      @alignItems='center'
+      local-class='file-compare-header-project-overview'
+      data-test-fileCompareHeader-projectOverview
+    >
+      <AkStack @width='full' @alignItems='center' {{style gap='0.6429em'}}>
+        <AkStack @alignItems='center' {{style gap='0.4286em'}}>
```

Comment:
please see if we can use standard gap values

Reply:
Resolved